### PR TITLE
RSDK-3292 - fix example python movement sensors

### PR DESCRIPTION
**Pull request by jckras**
Follow-up work to be done: https://viam.atlassian.net/browse/RSDK-3403

### CODE REVIEW

File: tests/mocks/components.py [hunk 1]
```diff
@@ -697,6 +697,7 @@ async def do_command(self, command: Mapping[str, ValueTypes], *, timeout: Option
         return {"command": command}
 
 
+
```
Comment (stuqdog) [inline tests/mocks/components.py:700]: (nit) probably we should get rid of this newline

File: tests/test_robot.py [hunk 2]
```diff
@@ -50,12 +51,13 @@
...
 from viam.utils import dict_to_struct, message_to_struct, struct_to_message
 
-from .mocks.components import MockArm, MockCamera, MockMotor, MockSensor
+from .mocks.components import MockArm, MockCamera, MockMotor, MockMovementSensor, MockSensor
 
 RESOURCE_NAMES = [
     ResourceName(namespace="rdk", type="component", subtype="arm", name="arm1"),
     ResourceName(namespace="rdk", type="component", subtype="camera", name="camera1"),
     ResourceName(namespace="rdk", type="component", subtype="motor", name="motor1"),
+    # ResourceName(namespace="rdk", type="component", subtype="sensor", name="movement_sensor1"),
```
Comment (stuqdog) [inline tests/test_robot.py:60]: As a general policy, I think it's best to not have commented code committed. It is often unclear what purpose it serves/used to serve, and there's no validation that the code still works as expected. Probably best to get rid of this entirely.

**Review by stuqdog — CHANGES_REQUESTED** [review]
A couple small suggestions, but otherwise this looks great! thanks for putting it together :)

**Comment by jckras** [comment]
Re-requested review after updating the tests to account for the dedupe done in generate metadata. Also added dedupe in generateStatus

File: tests/test_robot.py [hunk 3]
```diff
@@ -171,14 +196,15 @@ async def test_resource_names(self, service: RobotService):
...
+            assert len(response.resources) == len(RESOURCE_NAMES)
+            assert set(response.resources) == set(RESOURCE_NAMES)
 
     @pytest.mark.asyncio
     async def test_get_status(self, service: RobotService):
         async with ChannelFor([service]) as channel:
             client = RobotServiceStub(channel)
             response: GetStatusResponse = await client.GetStatus(GetStatusRequest())
-            assert list(response.status) == STATUSES
+            assert list(response.status) == (STATUSES)
```
Comment (cheukt) [inline tests/test_robot.py:207]: extra parenthesis?

**Review by cheukt — APPROVED** [review]
Looks great! small nit

**Review by stuqdog — APPROVED** [review]
LGTM!

**Pull request merged**